packages/react-components/src/components/ImageOverlay.tsx

```diff
@@ -139,7 +137,7 @@ export const ImageOverlay = (props: ImageOverlayProps): JSX.Element => {
             iconProps={cancelIcon}
             className={mergeStyles(closeButtonStyles(overlayTheme))}
             onClick={onDismiss}
-            ariaLabel={localeStrings.dismissButtonAriaLabel}
+            aria-label={localeStrings.dismissButtonAriaLabel}
```

Comment:
Good catch. All our 21 other IconButton is using `ariaLabel` instead of `aria-lable`. We might need to create a task to update all of them.